src/com/company/SquareArea.java

```diff
@@ -0,0 +1,63 @@
+package com.company;
+
+import java.util.Scanner;
+
+/**
+ * displayes the area of a square with the lenght provided by customer
+ * @author Marius Cristea
+ * @version 1
+ */
+public class SquareArea {
+    String squareLen;
+
+    public SquareArea(){
+        squareLen = createString();
+        calculateSquareArea(squareLen);
+    }
+
+    public static void main (String[] args){
+
+    }
+    /**
+     * displays the area of a square
+     * @param a lenght of the square
+     */
+    public static void calculateSquareArea(String a){
+        float len = Float.parseFloat(a);
+        if (len < 0){
+          System.out.println("In real life the square length may not be negative value!");
+        }
+        System.out.println("Square area with length " + len + " is = " + (len * len));
+    }
+    /**
+     *return a string inserted by the user and checks if the string holds an float number
+     * @return a string that has a value convertable to float
+     */
+    public static String createString() {
```

Comment:
you could reuse these kind of methods.
You can create another class, KeyboardIO for example and you could have static methods there thare readFloat or readInt or any other thing and they can do your custom logic, and you can use them from any other class in your project. So you don't need to copy paste it from class to class.